pegasus/sites.v3/code.org/public/css/careers.css

```diff
@@ -13,6 +13,7 @@
   display: flex;
   margin-bottom: 32px;
   padding: 18px 24px;
+  gap: 1em;
```

Comment:
Updates the spacing between icon and text on this banner:
<img width="350" alt="Screenshot 2023-05-18 at 4 01 29 PM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/a3d5d0f4-2222-43a8-a293-86b0132c30cd">